packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -187,7 +187,8 @@ describe('Withdraw ERC20 Token', () => {
   context('user has 0 L2-ERC20 and is on wrong chain', () => {})
   // TODO
   context(
-    'user has some LPT tokens which require token approval permission and is on L2',
+    'user has ERC20 tokens which require token approval permission and is on L2',
+    //eg. LPT tokens
```

Comment:
Small comment change here to address Fionna's concern in a previous PR.

Reply:
no this one should be kept as LPT
only LPT requires approval on L2
but all ERC20 that the user is depositing for the first time requires approval on L1
Dragiša has made the change to that test case description already